langchain4j-redis/src/test/java/dev/langchain4j/store/embedding/redis/RedisChatMemoryStoreIT.java

```diff
@@ -0,0 +1,146 @@
+package dev.langchain4j.store.embedding.redis;
+
+import com.redis.testcontainers.RedisContainer;
+import dev.langchain4j.data.message.ChatMessage;
+import dev.langchain4j.data.message.Content;
+import dev.langchain4j.data.message.ImageContent;
+import dev.langchain4j.data.message.SystemMessage;
+import dev.langchain4j.data.message.UserMessage;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.redis.testcontainers.RedisStackContainer.DEFAULT_IMAGE_NAME;
+import static com.redis.testcontainers.RedisStackContainer.DEFAULT_TAG;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class RedisChatMemoryStoreIT {
+
+    static RedisContainer redis = new RedisContainer(DEFAULT_IMAGE_NAME.withTag(DEFAULT_TAG));
+
+    private final String userId = "someUserId";
+    private RedisChatMemoryStore memoryStore;
+
+    @BeforeAll
+    static void beforeAll() {
+        redis.start();
+    }
+
+    @AfterAll
+    static void afterAll() {
+        redis.stop();
+    }
+
+    @BeforeEach
+    void setUp() {
+        this.memoryStore = RedisChatMemoryStore.builder()
+                .port(redis.getFirstMappedPort())
+                .host(redis.getHost())
+                .build();
+        memoryStore.deleteMessages(userId);
+        List<ChatMessage> messages = memoryStore.getMessages(userId);
+        assertThat(messages).isEmpty();
+    }
+
```

Comment:
tests for the constructor logic when user is null?

Reply:
done, for password too

Reply:
good call!